src/moi.jl

```diff
@@ -289,6 +289,88 @@ end
 accept_vector_set(::ProductMode{T}, ::Complement) where {T} = nothing
 accept_vector_set(::MixedMode{T}, ::Complement) where {T} = nothing
 
+function get_variable_complement(primal_model, dual_model, primal_con, dual_con)
+    return error(
+        "An internal error with variable complements occurred. Likely, your problem type does not yet support consideration of constrained variables.",
+    )
+end
+
+function get_variable_complement(
+    primal_model,
+    dual_model,
+    primal_con::MOI.ConstraintIndex{Fp,Sp},
+    dual_con::MOI.ConstraintIndex{Fd,Sd},
+) where {
+    Fp<:MOI.VariableIndex,
+    Sp<:Union{MOI.LessThan{T},MOI.GreaterThan{T}},
```

Comment:
SCALAR_SETS ?

Reply:
Would prefer this to throw an error in case of equalities (i.e. if something unexpected happens). 
Could also go with SCALAR_SETS for code readability in case you prefer and do @assert inside the function?